README.md

```diff
@@ -88,11 +88,18 @@ rustflags = [
 ]
 ```
 
-NOTE that if you were using GNU `ld` or GNU `gcc` to link your program then this
-won't work. Support for other linkers is being tracked in [issue #1]
+NOTE that if you were using GNU `ld` or GNU `gcc` to link your program then this won't work. Support for other linkers is being tracked in [issue #1].
 
 [issue #1]: https://github.com/knurling-rs/flip-link/issues/1
 
+## Testing
+
+Our CI enforces various checks. You can run them locally to make sure your PR will pass the CI:
+* `cargo fmt -- --check`
```

Comment:
Very minor nitpick (& probalby irrelevant), the command executed is `cargo fmt --all -- --check`.

Reply:
```suggestion
* `cargo fmt --all -- --check`
```